src/CMakeLists.txt

```diff
@@ -33,10 +33,13 @@ IF(ENABLE_TRACE)
 ENDIF()
 ADD_FEATURE_INFO(tracing GIT_TRACE "tracing support")
 
-CHECK_FUNCTION_EXISTS(futimens HAVE_FUTIMENS)
+enable_warnings(error)
+CHECK_SYMBOL_EXISTS(futimens sys/stat.h HAVE_FUTIMENS)
+# CHECK_FUNCTION_EXISTS(futimens HAVE_FUTIMENS)
```

Comment:
Let's remove this entirely, not comment it out.

Reply:
Could we rely on path "sys/stat.h" on all platforms? I guess it is better to add if (Apple) guard here.

Reply:
Yes, I agree - I broke this out into a separate file in #5962 - can you please double-check that this still works for you?  It looks like it works for me... 🤔

Reply:
It works as expected.